src/registry.c

```diff
@@ -742,6 +742,10 @@ parse_config_item(struct rxkb_context *ctx,
             if (!*name || !strlen(*name))  {
```

Comment:
should be fine, the dtd doesn't allow for more than one name. Arguably, this condition shouldn't even be seen because we fail the dtd validation (`name` is required). But not having a name and getting past the DTD would blow things up later - leaking memory is too niche to worry about for this case, IMO.

Reply:
Yeah, agreed.